lib/harness/renderer/file.ex

```diff
@@ -110,6 +124,33 @@ defmodule Harness.Renderer.File do
     end
   end
 
+  def generate(%Run{} = run, %__MODULE__{type: :hard_link} = link) do
+    to = Path.join(run.output_directory, link.output_path)
+    from = link.source_path
+
+    # There's no apparent reliable way to determine if an existing file at `to`
+    # is the hard link previously made by harness. Therefore, harness will
+    # always delete the `to` file and recreate the hard link. The `skip_files`
+    # config is available if the user wants to customize the contents of the
+    # `to` file.
+    #
+    # (We could infer that the desired hard link already exists if the contents
+    # of the two files are identical and the number of links reported by
+    # `File.stat` is 2, but that's not entirely reliable, so doesn't seem
+    # worthwhile since always re-linking is ok.)
+    with {:ok, %{type: :regular}} <- File.stat(to) do
```

Comment:
credo will probably tell you that this should be a `case` instead of a `with` because there's only one clause

up to you though, I don't totally buy in to that readability lint

Reply:
yeah, this is exactly why I re-genned .credo.exs and then set that one to false